xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -1,17 +1,22 @@
 """High-level reliable submission methods with XRPL transactions."""
 
 import asyncio
+from typing import Optional, Union
 
 from typing_extensions import Final
 
 from xrpl.asyncio.clients import Client
 from xrpl.asyncio.ledger import get_latest_validated_ledger_sequence
-from xrpl.asyncio.transaction.main import submit
+from xrpl.asyncio.transaction.main import _check_fee
```

Comment:
`_check_fee` should probably have its underscore removed - it's being imported somewhere, which means it's no longer a private function.

Reply:
If I changed it to `check_fee`, there would be a confusion with the `check_fee` boolean causing an error.

Reply:
If I make the change there will be a confusion with `check_fee` boolean when the function is called, causing an error.